Source/WebInspectorUI/UserInterface/Views/StorageSidebarPanel.js

```diff
@@ -340,7 +340,7 @@ WI.StorageSidebarPanel = class StorageSidebarPanel extends WI.NavigationSidebarP
             previousOnlyChild.flattened = false;
             this.contentTreeOutline.removeChild(previousOnlyChild);
 
-            var folderElement = new WI.FolderTreeElement(folderName);
+            var folderElement = new WI.FolderTreeElement(folderName, null, folderId);
```

Comment:
Help others quickly figure out what `null` is. This is a use case for `const` ;)
Also, please use `let` instead of `var` ([coding style guide](https://trac.webkit.org/wiki/WebInspectorCodingStyleGuide#Tokensspacingindentationsyntax)).

```suggestion
            const representedObject = null;
            let folderElement = new WI.FolderTreeElement(folderName, representedObject, folderId);
```